docs/guides/flux.md

```diff
@@ -110,35 +110,69 @@ So we now have an action called `userValueUpdated`, which when called dispatches
 ### Connecting a React Component to our Flux Store
 
 The store is now updating its data - but we have no React components connected to the store! Lets set one up:
+Here is an example of how to set up a connected Component. It shows several ways that a component can connect to Stores or simply be overridden with props.
 
 ```js
 // ./src/views/user/index.js
 
 import React from 'react';
 import { connect } from 'utils/flux';
-import Textbox from 'carbon-react/lib/components/textbox';
+import AccountInvitation from 'test/lib/components/accountinvitation';
 import UserStore from 'stores/user';
+import GiftsStore from 'stores/gifts';
 import UserActions from 'actions/user';
 
-class UserView extends React.Component {
+class AccountInvitation extends React.Component {
```

Comment:
Lets keep with the UserView

Reply:
Also, this can be a functional component, which really highlights how `connect` can be leveraged: lift state up into a function like `mapStateToProps` or higher in the component tree